### PR TITLE
fix #14 send post data to backend  when request method is post 

### LINKED ISSUE #14: $_POST not populated on HTTP POST
I've noticed that `$_POST` is not populated when sending fields (using HTTP POST) to a simple function given the sample template.

Related to #13, what is the specific aim of this project? If we are aiming to provide a fully functioning web-server, then this should be supported. If not, then this perhaps should be down to the implementing developer to expect some payload and parse it appropriately.

For example, developers from a REST background would expect a JSON body within a POST, but some PHP apps use `application/x-www-form-urlencoded` via HTML forms. 

I see in `bootstrap` the body of the request is being sent to the PHP webserver, however, I can't see it with `var_dump(file_get_contents('php://input'));` - Any Ideas? We could then provide a snippet of processing for JSON or Form Data.

### PR DESCRIPTION
**Pull request by cm-iwata**
bootstrap must  post data to backend when request method is post

### CODE REVIEW

File: bootstrap [hunk 1]
```diff
@@ -156,6 +156,9 @@ while (true) {
   }
 
   if (strlen($body) > 0) {
+    if($event['httpMethod'] === 'POST'){
```
Comment (txase) [inline bootstrap:159]: I think this works if the content type is `application/x-www-form-urlencoded`. But I'm not sure it works if the body is instead `application/json`. The docs make it sound like the body will be incompatible with what `CURLOPT_POSTFIELDS` expects (see http://php.net/manual/en/function.curl-setopt.php).

Can you verify what happens if you post a JSON payload with `Content-Type: application/json` set? If this change breaks posting non-form-urlencoded bodies, then please add a condition here that checks the content type.

**Review by txase — COMMENTED** [review]
Thanks for figuring this out! I don't really understand why setting the CURLOPT_READFUNCTION doesn't take care of this already, but oh well.

Please test non-form-encoded POST bodies to be sure this doesn't break them. Then this looks ready to be merged!

File: bootstrap (hunk 1 above)
Comment (cm-iwata) [reply bootstrap:159]: OK, I verified some test case.

## Verification code
```
<?php
$input = file_get_contents("php://input");
echo $input;
```

## verify at local machie

### prepare builtin server
```
php -S localhost:8080 index.php
```

### send request
```
 curl -X POST http://localhost:8080 -H 'Content-Type: application/json' -H 'cache-control: no-cache' -d '{"key": "val"}'
```

### result
```
{"key": "val"}
```

## verify at AWS using arn:aws:lambda:${AWS::Region}:887080169480:layer:php71:5

### send request
```
 curl -X POST https://xxxxx.execute-api.ap-northeast-1.amazonaws.com/Prod/index.php -H 'Content-Type: application/json' -H 'cache-control: no-cache' -d '{"key": "val"}'
```

### result
```
```
empty response body returned 


## verify at AWS using my layer that patched the issue 

### send request
```
 curl -X POST https://xxxxx.execute-api.ap-northeast-1.amazonaws.com/Prod/index.php -H 'Content-Type: application/json' -H 'cache-control: no-cache' -d '{"key": "val"}'
```

### result
```
{"key": "val"}
```


I think, It was not send Json Payload from before


I'm not good at English.

Comment (txase) [reply bootstrap:159]: Awesome! Looks great, thanks for confirming the body still comes through!

**Review by txase — APPROVED** [review]

**Comment by txase** [comment]
This fix is now published in version 6 of the layer!

🙏 @cm-iwata!

**Pull request merged**